uint8 message_type
string manager_id
string node_id
uint8 return_value
string package_name
string node_name
string param_json
string remap_json